hadoop-ssm-project/src/test/java/org/apache/hadoop/ssm/sql/DBTest.java

```diff
@@ -28,12 +28,14 @@
   private static final String DB_PATH = "/tmp/test.db";
 
   @Before
-  public void setUp() throws ClassNotFoundException {
+  public void setUp() throws Exception {
     File db = new File(DB_PATH);
```

Comment:
better to use TestDBUtil.getUniqueDBFilePath() to get a db file path for testing